packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -158,6 +255,11 @@ export class AtomicResultList implements InitializableComponent {
       return <atomic-result-list-placeholder></atomic-result-list-placeholder>;
     }
 
-    return [this.templateHasError && <slot></slot>, this.results];
+    return (
+      <div class={`list-root ${this.getClasses().join(' ')}`}>
+        {this.templateHasError && <slot></slot>}
```

Comment:
What was `<slot></slot>` for? Was it just to display nothing when there's an error? If so, I'll remove it.

Reply:
If the templates had errors, they would render an atomic-component-error, to display that error we render the slot

Reply:
Feel free to manage errors differently